README.md

```diff
@@ -1,2 +1,243 @@
-# blockstack-ng
-Reference implementation of the Stacks blockchain v2
+# Stacks 2.0
+
+Reference implementation of https://blockstack.org/whitepaper.pdf in Rust.
+
+[![CircleCI](https://circleci.com/gh/blockstack/blockstack-core/tree/develop.svg?style=svg)](https://circleci.com/gh/blockstack/blockstack-core/tree/develop)
+
+## Repository
+
+| Blockstack Topic/Tech | Where to learn more more |
+|---------------------------------|------------------------------------------------------------------------------|
+| Stacks 2.0 | [master branch](https://github.com/blockstack/blockstack-core/tree/master) |
+| Stacks 1.0 | [legacy branch](https://github.com/blockstack/blockstack-core/tree/develop) |
+| Use the package | [our core docs](https://docs.blockstack.org/core/naming/introduction.html) |
+| Develop a Blockstack App | [our developer docs](https://docs.blockstack.org/browser/hello-blockstack.html) |
+| Use a Blockstack App | [our browser docs](https://docs.blockstack.org/browser/browser-introduction.html) |
+| Blockstack the company | [our website](https://blockstack.org) |
+
+## Design Thesis
+
+Stacks 2.0 is an open-membership replicated state machine produced by the coordination of a non-enumerable set of peers.
+
+To unpack this definition:
+
+- A replicated state machine is two or more copies (“replicas”) of a given set of rules (a “machine”) that, in processing a common input (such as the same sequence of transactions), will arrive at the same configuration (“state”).  Bitcoin is a replicated state machine — its state is the set of UTXOs, which each peer has a full copy of, and given a block, all peers will independently calculate the same new UTXO set from the existing one.
+- Open-membership means that any host on the Internet can join the blockchain and independently calculate the same full replica as all other peers.
+- Non-enumerable means that the set of peers that are producing the blocks don’t know about one another — they don’t know their identities, or even how many exist and are online.  They are indistinguishable.
+
+## Roadmap
+
+- [x] [SIP 001: Burn Election](https://github.com/blockstack/blockstack-core/blob/develop/sip/sip-001-burn-election.md)
+- [x] [SIP 002: Clarity, a language for predictable smart contracts](https://github.com/blockstack/blockstack-core/blob/develop/sip/sip-002-smart-contract-language.md)
+- [x] [SIP 004: Cryptographic Committment to Materialized Views](https://github.com/blockstack/blockstack-core/blob/develop/sip/sip-004-materialized-view.md)
+- [x] SIP 005: Blocks, Transactions, and Accounts
```

Comment:
I think SIP 005 is ready, if you all want to take a look at the SIP 005 PR.  Then we can link it here :pray:

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1251/commits/58690348304e8c9c91e76132e25a06db443883d1